.github/workflows/ci_automation_hub_collection.yml

```diff
@@ -33,22 +33,25 @@ jobs:
       - name: install ansible
         run: pip3 install ansible
 
-      - name: Checkout the galaxy collection
+      - name: Checkout the ansible_hub collection
         uses: actions/checkout@v4
         with:
-          repository: ansible/galaxy_collection
-          path: galaxy_collection
+          repository: ansible-collections/ansible_hub
+          path: ansible_hub
+
+      - name: copy galaxy.yml.j2 to ansible_hub
+        run: |
+        cp .github/files/galaxy.yml.j2 /ansible_hub
```

Comment:
indentation error
```suggestion
        run: |
            cp .github/files/galaxy.yml.j2 /ansible_hub
```

Reply:
Thanks!  I thought it was the paths and overlooked the obvious...  :smile: